qiskit/test/mock/fake_openpulse_2q.py

```diff
@@ -89,10 +89,11 @@ def __init__(self):
                                        PulseQobjInstruction(name='test_pulse_1', ch='d1', t0=20),
                                        PulseQobjInstruction(name='fc', ch='d1',
                                                             t0=20, phase=2.1)]),
-                     Command(name='measure', qubits=[0],
+                     Command(name='measure', qubits=[0, 1],
```

Comment:
Update to match what `meas_map` claimed

Reply:
Yes, the above works as it is written. (Did I misunderstand the question? Why would you need two measure commands?)